ДЗ1

blackHATred opened the pull request:
Выполнил свою часть, протестировал разделы Авторизации и Регистрации, Профиль пользователя, Рецензии на фильмы и сериалы

blackHATred committed:
Create Cybercutlets.md

blackHATred committed:
ДЗ1. Регистрация и авторизация

blackHATred committed:
ДЗ1. Профиль пользователя

blackHATred committed:
ДЗ1. Рецензии

KristinaBu committed:
Update Cybercutlets.md

KristinaBu committed:
Update Cybercutlets.md

KristinaBu committed:
Update Cybercutlets.md

KristinaBu commented on Cybercutlets.md:
```diff
@@ -0,0 +1,205 @@
+  <img width="551" alt="image" src="https://github.com/user-attachments/assets/52c33529-226f-474c-a4e1-d6683510a413">
+- При проблемах на стороне клиента \ сервера при публикации \ редактировании рецензии появляется характерное сообщение об ошибке  
+  <img width="560" alt="image" src="https://github.com/user-attachments/assets/41712337-45bd-4c01-93d5-8bd00b58a6e3">
+
```
Нужно добавить проверку того, что оценка комментариев влияет на рейтинг фильма (и правильно ли влияет)

blackHATred committed:
корректировки из pull request

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+
+![image](https://github.com/user-attachments/assets/6e8227b1-3a76-473b-bc14-74d3f87aacf7)
+
+- При вводе названия фильма выводятся варианты фильмов, сериалов и персон:
```
По какому принципу должны выводится варианты? Наверное стоит сформулировать критерий того, что должно выводится при вводе названия

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+![image](https://github.com/user-attachments/assets/df8c2255-d69b-4c5c-abcf-69836ba24f56)
+![image](https://github.com/user-attachments/assets/95ccc8a5-0039-49cf-9bf3-9fec683ce390)
+
+- при нажатии на персону или сериал/фильм блоке результатов поиска, открываются их страницы. Данные на них соответствуют выведенным в поиске данным.
```
Поправить орфографию

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+
+- при нажатии на персону или сериал/фильм блоке результатов поиска, открываются их страницы. Данные на них соответствуют выведенным в поиске данным.
+
+- При вводе неполного названия фильма/сериала или персоны поиск выводит результаты, которые содержат в себе указанную строку или похожи на нее:
```
Вот здесь критерии есть. Стоит продублировать их в месте, где я оставил ревью выше

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+    ![image](https://github.com/user-attachments/assets/ad2cb2fc-62f5-42a1-a05d-4b9bdc78ad90)
+    ![image](https://github.com/user-attachments/assets/412e6263-3692-439f-bc77-b73dd9c2875b)
+
+- При вводе большого числа символов в поисковую строку поиск выводит предыдущий результат - это неверное поведение:
```
Неверное поведение помечай как у меня, чтобы документ был в едином стиле. Сами пункты должны формулироваться в утвердительном тоне. Т.е. должно быть "При вводе большого числа символов в поисковую строку поиск отображает ошибку пользователю с характерным сообщением" или что-то похожее. Главное, чтобы утверждение содержало то поведение, которое ты ожидаешь, а не которое есть на самом деле

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+- Дополнение к предыдущему: при вводе очень большого числа символов поиск одновременно выводит строку об отсутствии результатов и показывает спиннер загрузки результатов. При повторном открытии поиска спиннер не исчезает.
+![image](https://github.com/user-attachments/assets/c0f22696-2b39-45eb-8563-86907394e26e)
+![image](https://github.com/user-attachments/assets/7740e56c-1fff-4a8c-8ca2-382213a3403d)
+> Итог: оформить репорт об органичении длины вводимой строки
```
целиком засунуть под ">" в Readme, если ты это оформляешь как найденный баг

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+
+- Если найти фильм из раздела "Календарь релизов", то у него нет рейтинга. В разделе поиска рейтинг отображается, он равен "0.0", хотя на странице фильма из "Календаря релизов" рейтинга нет, так как фильм еще не вышел - это баг.
+
+![image](https://github.com/user-attachments/assets/b9e64d78-7e35-4215-b8c6-a9f1ca309082)
```
аналогично оформить баг как следует. И переписать формулировку под поведение, которое ожидаешь, а не которое получила

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+
+  ![image](https://github.com/user-attachments/assets/45207797-9c14-451f-85a7-cd4ff385b94e)
+
+- При переключении на типы подборок все данные соответствуют по теме каждому выбранному типу - фильтрация без повторений.  
```
Что такое "фильтрация без повторений"? Размытые формулировки не приветствуются

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+
+![image](https://github.com/user-attachments/assets/0fe4c228-16b8-4be9-8f7b-c1667e968cea)
+
+- При нажатии на заявку открывается ее страница. Над списком контента (фильмов/сериалов) отображается название подборки  
```
Заявку? Это что?

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+
+![image](https://github.com/user-attachments/assets/ba4f569f-59f5-41ea-ac72-cc0ec75a16b9)
+
+- При нажатии на кнопку добавления в избранное ее вид меняется, также загорается звуздочка:
```
Опечатки. А еще ты не указала, что должна смениться надпись на кнопке

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+
+![image](https://github.com/user-attachments/assets/eed4bfb4-f404-4570-8b86-b1d12b0d85fc)
+
+- Находясь в избранном, можно удалить оттуда фильм. При этом со страницы самого фильма отметка "В избранном" пропадает, как и должно быть.
```
"как и должно быть" убрать. Ты и так описываешь ожидаемое поведение. Всё, что ты пишешь, должно соответствовать "как и должно быть"

blackHATred commented on Cybercutlets.md:
```diff
@@ -0,0 +1,211 @@
+
+![image](https://github.com/user-attachments/assets/297fceac-7a62-445c-ba6e-992f42267dfc)
+
+- Также можно удалить из избранного со страницы фильма, нажав на кнопку "в избранном", тогда название кнопки обратно сменится на "добавить в избранное", со страницы фильма этот фильм также удалится.
```
Слова "также можно" не соответствуют критерию сухих формулировок

blackHATred reviewed (commented):
Оставил свои комментарии

KristinaBu committed:
Update Cybercutlets.md

KristinaBu committed:
Update Cybercutlets.md

KristinaBu committed:
Update Cybercutlets.md

KristinaBu committed:
Update Cybercutlets.md

blackHATred committed:
Правки

blackHATred committed:
фикс поехавшей картинки

blackHATred committed:
фикс картинок

blackHATred committed:
правки

Улучшил разбиение по разделам;
Ссылки на разделы иного вида;
Декомпозировал некоторые проверки.

blackHATred committed:
правки профиля

Декомпозировал пункты в разделе профиля;
Внёс уточнения в разделе профиля;
Добавил требования к виду блока с рецензиями пользователя.

KristinaBu committed:
Update Cybercutlets.md

blackHATred committed:
Update Cybercutlets.md

дополнительное разбиение кейсов на странице регистрации по почте, паролю и повтороному вводу пароля
